docs/content/home/contribution.md

```diff
@@ -0,0 +1,40 @@
+# Contribution
+
+If you want to help us to make everything related to UI testing easier to others, you can contact us directly.
+
+[Alexey Bykov](https://github.com/nonews) <br>
+[Dmitriy Movchan](https://github.com/v1sar) <br>
+[Sergio Sastre](https://github.com/sergio-sastre) <br>
+
+## Companies experience template
+
+Please, use this template to share your tooling with others
+
+```markdown
+## Company name
+
+`UI testing`
+
+:   
+`Write:` [Framework](Link to the github/documantation) <br>
+`Who:` [Android Engineers, QA Automation]<br>
+`Runner:` [Runner](Link to the github/documantation) <br>
```

Comment:
Maybe **Orchestrator** as well? I am very against it since it  considerably slows down tests but...
And what about test **sharding or pools** (like fork/Shazam, or the brand new TestingUnitPlatform and Gradle Managed devices from google)?

Reply:
It's just an example, what can we have inside the adoption section :) 

About test runner, there a separate page:
https://android-ui-testing.github.io/Cookbook/practices/test_runners_review/